flash_examples/integrations/baal/image_classification_active_learning.py

```diff
@@ -51,9 +50,10 @@
 
 # 4. Predict what's on a few images! ants or bees?
 datamodule = ImageClassificationData.from_files(
-    predict_files=["data/hymenoptera_data/val/bees/65038344_52a45d090d.jpg"]
+    predict_files=["data/hymenoptera_data/val/bees/65038344_52a45d090d.jpg"],
+    batch_size=1,
 )
-predictions = trainer.predict(model, datamodule=datamodule)
+predictions = flash.Trainer().predict(model, datamodule=datamodule)
```

Comment:
Is there an issue with reusing a trainer?

Reply:
I was debugging the data_pipeline_state was containing everything.